src/aws-app/main.ts

```diff
@@ -22,13 +22,20 @@ export default async () => {
 
 	const getRateLimiter = createRateLimiterFactory(config['call-rate-rps'], new AWSRateLimitExhaustionRetryStrategy())
 
+	const tagsRateLimiterFactory = createRateLimiterFactory(
+		config['call-rate-rps'],
+		new AWSRateLimitExhaustionRetryStrategy(),
+	)
+	const tagsRateLimiter = tagsRateLimiterFactory('tags')
```

Comment:
I think we can still use `getRateLimiter` to create a new rate limiter for the tags. getRateLimiter returns a new independent rate limiter for each unique service name. So what we could also do:

```ts
const tagsRateLimiter = getRateLimiter('resource-groups-tagging')
```

I strongly recommend implementing this within the `getAwsScanners` function, as the function has all the necessary dependencies.

Reply:
Good catch... kinda overlooked this. Fixed now!